test_examples.sh

```diff
@@ -12,7 +12,7 @@ function ncores {
   fi
 }
 
-export OPTS="prune\nprune_no_hoist\nhoist_osr\nconst_fold\nhoist_assign\nhoist_drop\nmin_live\ninline_small\ninline_med"
+export OPTS="prune\nprune_no_hoist\nassumptions\nhoist_osr\nconst_fold\nhoist_assign\nhoist_drop\nmin_live\ninline_small\ninline_med"
```

Comment:
If we're adding this to the test runner, maybe we should have add a short program to the `examples` directory? The one in `tests.ml` should be enough.